test/functional/wallet_listsinceblock.py

```diff
@@ -445,6 +446,17 @@ def test_send_to_self(self):
         assert any(c["address"] == addr for c in coins)
         assert all(self.nodes[2].getaddressinfo(c["address"])["ischange"] for c in coins)
 
+    def test_label(self):
+        self.log.info("Test label")
+        new_addr = self.nodes[1].getnewaddress("new_addr", "bech32")
+
+        self.nodes[2].sendtoaddress(new_addr, "0.001")
+        block_hash = self.generate(self.nodes[2], 1)[0]
+
+        new_addr_transactions = self.nodes[1].listsinceblock(label="new_addr", blockhash=block_hash)["transactions"]
+        for transaction in new_addr_transactions:
+            assert_equal(transaction["label"], "new_addr")
+
```

Comment:
If the wildcard behavior is kept, because it is the same as `listtransactions`, maybe add a test that verifies that wildcard and the omitted label filter behave the same. Also test with empty label.